extension/src/prototypes/view/containers/KeyboardHandlers.ts

```diff
@@ -0,0 +1,138 @@
+import { useCallback, useRef, useState } from "react";
+
+import { useConstant, useWindowEvent } from "../../react-helpers";
+
+const directions = ["forward", "backward", "right", "left", "up", "down"] as const;
+
+type Direction = (typeof directions)[number];
+
+function isDirection(value: unknown): value is Direction {
+  return directions.includes(value as Direction);
+}
+
+const modes = ["sprint"] as const;
+
+type Mode = (typeof modes)[number];
+
+function isMode(value: unknown): value is Mode {
+  return modes.includes(value as Mode);
+}
+
+type KeyAssignments = Record<string, Direction | Mode>;
+
+const defaultKeyAssignments: KeyAssignments = {
+  KeyW: "forward",
+  KeyS: "backward",
+  KeyA: "left",
+  KeyD: "right",
+  Space: "up",
+  ControlLeft: "down",
+  ShiftLeft: "sprint",
+};
+
+export const KeyboardHandlers = () => {
```

Comment:
It seems ScreenSpaceCamera isn't working. When I move camera then camera should be rotated.

https://github.com/eukarya-inc/PLATEAU-VIEW-3.0/assets/34934510/1cdf875e-3501-407d-b2d9-d81ca923a306

Reply:
This should be fixed.